code/datums/ai_laws.dm

```diff
@@ -78,6 +78,7 @@
 		src.inherent.len = number
 
 	src.inherent[number] = law
+	statlog_ailaws(1, law, "Ion Storm")
```

Comment:
not necessarily due to an ion storm...

Reply:
There are currently no other functions that call this, and the `statlog_ailaws` code assumes anything that isn't a mob is an ion storm anyway.

Reply:
still, the function you are modifying is called `replace_inherent_law` not `replace_inherent_law_due_to_ionstorm`

this is a maintainability concern